SU2_CFD/src/drivers/CDiscAdjMultizoneDriver.cpp

```diff
@@ -217,6 +229,96 @@ void CDiscAdjMultizoneDriver::StartSolver() {
 
 }
 
+void CDiscAdjMultizoneDriver::TapeTest() {
+
+  if (rank == MASTER_NODE) {
+    cout <<"\n---------------------------- Start Debug Run ----------------------------" << endl;
+  }
+
+  int total_errors = 0;
+  struct AD::ErrorReport error_report;
```

Comment:
Please look for all the unnecessary `struct`
```suggestion
  AD::ErrorReport error_report;
```